end-to-end-tests/cypress/pages/checkAnswers.ts

```diff
@@ -1,26 +1,33 @@
-import { BasePage } from './basePage'
+import { BasePage } from './basePage';
 
 class CheckAnswersPage extends BasePage {
 
-  public slug = 'transfers/checkyouranswers'
+  public slug = 'transfers/checkyouranswers';
 
   public checkDetails(outgoingTrust, incomingTrust): this {
 
-    cy.get('.govuk-grid-column-full').as('trustAcademiesDetails')
+    cy.get('.govuk-grid-column-full').as('trustAcademiesDetails');
 
-    cy.get('@trustAcademiesDetails').should('contain.text', outgoingTrust.name)
-    cy.get('@trustAcademiesDetails').should('contain.text', outgoingTrust.ukPrn)
+    cy.get('@trustAcademiesDetails').then($el => {
+      const textContent = $el.text().trim();
+      
+      expect(textContent).to.contain(outgoingTrust.name.trim());
+      expect(textContent).to.contain(outgoingTrust.ukPrn.trim());
+      expect(textContent).to.contain(outgoingTrust.academies.trim());
+      expect(textContent).to.contain(incomingTrust.name.trim());
+      //expect(textContent).to.contain(incomingTrust.ukPrn.trim());
```

Comment:
remove comment if not needed

Reply:
This has been removed as it is not needed.